### PR TITLE
Add children into Modal to show custom elements

### LINKED ISSUE #146: Request: Add thumbnail slider on Lightbox
* What's your use case: I want to show the thumbnail slider below the current image. I try to create a new component separate with Lightbox to make that but something not working. I can't use the keyboard shortcut to make the move request because the thumbnail is not inside of the lightbox.

I think we just push some `props.children` show inside lightbox and it will work.

* What interface do you have in mind
Maybe like this
screen shot 2018-09-14 at 17 50 04

### PR DESCRIPTION
**Pull request by tungvn**
It maybe fixes #146 by leaving dev to create their custom thumbnail slider

### CODE REVIEW

**Review by The-Code-Monkey — APPROVED** [review]
This will work really well and will not break anything. plus sort of need this as a client just asked me to integrate this

**Comment by opr** [comment]
Can someone with the appropriate power merge this please? 😊😊

**Comment by forcewake** [comment]
Hi. Is it possible to merge this as it was approved.

File: src/react-image-lightbox.js [hunk 1]
```diff
@@ -1614,6 +1615,8 @@ class ReactImageLightbox extends Component {
             </div>
           )}
         </div>
+
```
Comment (wuweiweiwu) [inline src/react-image-lightbox.js]: Can you remove this new line?

**Review by wuweiweiwu — CHANGES_REQUESTED** [review]

**Comment by tungvn** [comment]
@The-Code-Monkey please merge it :)

**Review by Tornek — APPROVED** [review]

**Comment by mkamalkayani** [comment]
@wuweiweiwu What is preventing this pull request to be merged?
I would be happy to help.

File: src/react-image-lightbox.js (hunk 1 above)
Comment (andrew-giangrant) [reply src/react-image-lightbox.js]: Looks like someone else resolved this edit over a year ago. By the way, this is a really small nitpick for a PR that has been open for over 2 years.

**Review by agiangrant — APPROVED** [review]
Looks good to me

**Pull request closed without merging**